packages/reactivity/src/ref.ts

```diff
@@ -130,10 +195,50 @@ class RefImpl<T> {
   }
 }
 
+/**
+ * Execute any effects tied to a shallow ref manually.
```

Comment:
I think we can update this one to the docs version: https://vuejs.org/api/reactivity-advanced.html#triggerref